src/js/themes/base.js

```diff
@@ -1462,6 +1462,61 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         // extend: undefined,
       },
     },
+    tag: {
```

Comment:
overall looks good can we also add types to the `base.d.ts` file

Reply:
Good point. Thank!